src/content/docs/de/guides/markdown-content.mdx

```diff
@@ -560,3 +655,22 @@ Wenn du Prism verwenden willst, musst du ein Stylesheet zur Syntaxhervorhebung z
 4. Lade es im [`<head>`-Abschnitt deiner Seite](/de/basics/astro-pages/#seiten-html) mit einem `<link>`-Tag.
```

Comment:
In Zeile 652 würde glaube ich der Slug `#syntaxhervorhebung` der richtige sein. GitHub lässt mich aber aus irgendeinen Grund dort keinen Vorschlag erstellen :/